src/pages/settings/Subscription/SubscriptionPlan.tsx

```diff
@@ -0,0 +1,94 @@
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useSubscriptionPlan from '@hooks/useSubscriptionPlan';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import SaveWithExpensifyButton from './SaveWithExpensifyButton';
+
+function SubscriptionPlan() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+
+    const subscriptionPlan = useSubscriptionPlan();
+    const [privateSubscription] = useOnyx(ONYXKEYS.NVP_PRIVATE_SUBSCRIPTION);
+
+    const isCollect = subscriptionPlan === CONST.POLICY.TYPE.TEAM;
+    const isAnnual = privateSubscription?.type === CONST.SUBSCRIPTION.TYPE.ANNUAL;
+
+    const benefitsList = isCollect
+        ? [
+              translate('subscription.yourPlan.collect.benefit1'),
+              translate('subscription.yourPlan.collect.benefit2'),
+              translate('subscription.yourPlan.collect.benefit3'),
+              translate('subscription.yourPlan.collect.benefit4'),
+              translate('subscription.yourPlan.collect.benefit5'),
+              translate('subscription.yourPlan.collect.benefit6'),
+          ]
+        : [
+              translate('subscription.yourPlan.control.benefit1'),
+              translate('subscription.yourPlan.control.benefit2'),
+              translate('subscription.yourPlan.control.benefit3'),
+              translate('subscription.yourPlan.control.benefit4'),
+              translate('subscription.yourPlan.control.benefit5'),
+              translate('subscription.yourPlan.control.benefit6'),
+              translate('subscription.yourPlan.control.benefit7'),
+          ];
+
+    return (
+        <Section
+            title={translate('subscription.yourPlan.title')}
+            isCentralPane
+            titleStyles={styles.textStrong}
+        >
+            <View style={[styles.borderedContentCard, styles.mt5, styles.p5]}>
+                <Icon
+                    src={isCollect ? Illustrations.Mailbox : Illustrations.ShieldYellow}
+                    width={variables.iconHeader}
+                    height={variables.iconHeader}
+                />
+                <Text style={[styles.headerText, styles.mt2]}>{translate(`subscription.yourPlan.${isCollect ? 'collect' : 'control'}.title`)}</Text>
+                <Text style={[styles.textLabelSupporting, styles.mb2]}>
+                    {translate(`subscription.yourPlan.${isCollect ? 'collect' : 'control'}.${isAnnual ? 'priceAnnual' : 'pricePayPerUse'}`)}
```

Comment:
In addition to this text being variable based on collect vs control and annual vs PPU, we also need to make the price variable based on currency/location as detailed in the doc. We said we'd create a `getPreferredCurrency` function for that. Should we cover that in this PR or is there a more suitable GH for handling that scope?

Reply:
(Search for `public const PRICES` in https://gist.github.com/amyevans/c55a02141f1945da7309e09aaf263515 for the dynamic price to show based on each currency US/AU/GB/NZ)

Reply:
I thought about integrating this in the second phase (backend integration), as it depends on payment card currency. What do you think?

Reply:
Works for me!